### DISCUSSION TITLE
Laravel Livewire Support jQuery  plugin is possible or not

### DISCUSSION THREAD

**Discussion by wasitmirani** [livewire/livewire#1768] [4. General]
jQuery plugin supported or not please kindly information  about jquery plugin like select2 and other plugin of jquery using on livewire project

**Comment by detygon** [comment]
Livewire act as bridge between your backend and your UI. So you can still use whatever tools you're used to. You can watch [this screencast](https://laravel-livewire.com/screencasts/s6-refactor-pikaday-trix) by Caleb to see how to integrate a third party library in your component. Hope it helps, or try to provide more details about the issue you're encountering.
